packages/amplication-client/src/Layout/InnerTabLink.scss

```diff
@@ -18,6 +18,7 @@
 
   .amp-icon {
     font-size: var(--title3-font-size);
+    width: 22px;
```

Comment:
We recently aligned all CSS properties on the design-system, and you should not manually change the CSS of the icon.
Please try to remove the width and font-size properties from the CSS - and instead use the size property on the Icon component.

Reply:
Thanks for reviewing @yuval-hazaz, started working on it.